jibri/rootfs/defaults/jibri.conf

```diff
@@ -0,0 +1,58 @@
+jibri {
+    api {
+        http {
+            external-api-port = {{ .Env.JIBRI_HTTP_API_EXTERNAL_PORT | default 2222 }}
+            internal-api-port = {{ .Env.JIBRI_HTTP_API_INTERNAL_PORT | default 3333 }}
+        }
+        xmpp {
+            environments = [
+   		        {
+                    name = "prod environment"
```

Comment:
nit: Do other dockerfiles use strings like this as a default?  I think we may want to make this pull from a variable.